x-pack/plugins/security_solution/public/management/components/console/components/console_header.tsx

```diff
@@ -31,12 +31,13 @@ export const ConsoleHeader = memo<ConsoleHeaderProps>(({ TitleComponent }) => {
   }, [dispatch, isHelpOpen]);
 
   return (
-    <EuiFlexGroup gutterSize="none" alignItems="center">
-      <EuiFlexItem grow className="eui-textTruncate">
+    <EuiFlexGroup gutterSize="none" alignItems="center" justifyContent="spaceBetween">
+      <EuiFlexItem grow={1} className="eui-textTruncate">
         {TitleComponent ? <TitleComponent /> : ''}
       </EuiFlexItem>
-      <EuiFlexItem grow={false}>
+      <EuiFlexItem grow={1}>
```

Comment:
I'm not sure about these changes here (and above) with the `grow`, but we'll take a look the next time we touch the console's header (to add more buttons to the right side)